File: vimana source code/crypto-lib/Java/cryptolib/src/main/java/io/vimana/cryptolib/dataformat/AEAD.java
package io.vimana.cryptolib.dataformat;

/**
 * Class that represends decryption result of AEAD (unencrypted part) and encrypted part of message
 * @author alukin@gmail.com
 */
public class AEAD {
    /**
     * plain part of AEAD message
     */
    public byte[] plain;
    /**
     * decrypted part of AEAD message
     */
    public byte[] decrypted;
    /**
     * indicator of correctness
     */
    public boolean hmac_ok;
}

File: vimana source code/crypto-lib/JS/cryptolib/src/app/dataformat/AEAD.js
const AEAD = function() {
    /**
     * plain part of AEAD message
     */
    this.plain;
    /**
     * decrypted part of AEAD message
     */
    this.decrypted;
    /**
     * indicator of correctness
     */
    this.hmac_ok;
}

module.exports = AEAD;